### ISSUE TITLE
DittoMultiProcessorAttribute restricted to property only.

### ISSUE THREAD

**Issue by JimBobSquarePants** [leekelleher/umbraco-ditto#178]
Currently the `DittoMultiProcessorAttribute` is restricted to property decoration only which restricts its usefulness. I cant see why that would be the case. Is there something I have missed?

``` c#
[AttributeUsage(AttributeTargets.Property, AllowMultiple = true, Inherited = false)]
```

https://github.com/leekelleher/umbraco-ditto/blob/207c8d0233e509df29b5178059f7e97ccd129de6/src/Our.Umbraco.Ditto/ComponentModel/Attributes/DittoMultiProcessorAttribute.cs#L9

**Comment by mattbrailsford** [comment]
Nope, this was simply overlooked. Fixed now in commit  ec3e7c9

**Comment by JimBobSquarePants** [comment]
Sweet :) I think we should probably add that as a constant somewhere to give us a standard rule-set to draw from.
